app/src/main/res/layout/activity_main.xml

```diff
@@ -44,4 +54,16 @@
         app:backgroundTint="@android:color/holo_blue_dark"
         app:srcCompat="@android:drawable/stat_sys_data_bluetooth" />
 
+    <android.support.design.widget.FloatingActionButton
+        android:id="@+id/fab2"
```

Comment:
Can you make this name a bit more descriptive, and also the name of the other floating action button?